opal/include/opal/sys/arm64/atomic.h

```diff
@@ -117,7 +117,7 @@ static inline bool opal_atomic_compare_exchange_strong_rel_32(opal_atomic_int32_
     int32_t prev, tmp;
     bool ret;
 
-    __asm__ __volatile__("1:  ldxr    %w0, [%2]       \n"
+    __asm__ __volatile__("1:  ldaxr   %w0, [%2]       \n"
```

Comment:
Likewise, adding an acquire to a *strong_rel_32 function, doesn't seem right.

Reply:
Yes, you are right.  strong_rel no need to change, only  exchange_strong_ need change to stlxr.  I updated